Articles/Blog/2023-03-08-productivity-practices-and-tools.md

```diff
@@ -0,0 +1,157 @@
+
+* **Tasks**: identify something that needs to be done (e.g. a bug to fix).
+* **Observations**: describe something that happened worth remembering almost like in a lab journal.
+* **Ideas**: capture unspecific but maybe important thoughts to be processed later (e.g. a better approach to a sorting problem).
+* **Lessons**: record key takeaways worth sharing with others (even your future self).
+
+Effective  journaling also demands that one periodically slow down, pause, and reflect to review their record of activity.
+Such pauses enable one to make more deliberate decisions (e.g. acting instead of re-acting) about future work as well as to recognize patterns of activity worth changing.
+While there are many approaches to journaling, two useful sets of strategies come from [Bullet Journaling by Ryder Carroll](https://bulletjournal.com/) and [Building A Second Brain by Tiago Forte](https://www.buildingasecondbrain.com/)
+
+#### Key Journaling Strategies
+
+What to journal and how to journal are deeply personal.
+Here are a few of the high level strategies to consider:
+
+First, treat a journal as an append only log.
+Journaling is most useful when one can make entries quickly and easily.
+The more effort involved, the less likely it will be done.
+Now, one may ask, *"if the journal is append only, and everything is jumbled together, how can you ever find anything?"*
+The answer is in the second key strategy.
+
+Second, periodically reflect and distill what was journaled.
+Sort through the material to organize what is truly important and copy into more permanent homes.
+Bullet journaling recommends using pen and paper journals, because it requires effort to physically copy things from one place to another.
+
+Common reflection cadences are daily, weekly and monthly.
+Where appropriate, it might be worthwhile to align reflection cadence with a project's sprint cadence.
+
+[Building a Second Brain](https://www.buildingasecondbrain.com/) recommends *progressive summarization* which involves underlining important entries upon daily review, highlighting important underlined entries upon weekly review and migrating (copying) important highlighted entries upon monthly review.
+
+At longer cadences, more substantial reflection and review is appropriate.
+For example, at monthly intervals, its reasonable to check in with yourself and ask...what should I stop, start, and continue for next month?
+Are my practices productive and effective or do I need to make some adjustments?
+At longer intervals (e.g. seasonally), it is a good practice to ask yourself deeper questions...what is my vision/theme/aspiration(s) for this season?
+How well have I live up to my aspirations so far?
+At these more substantial reflection points, its often a good practice to bring the old journal to a close and start a *new* journal by copying key information (e.g. migrating) forward.
```

Comment:
```suggestion
At these more substantial reflection points, its often a good practice to bring the old journal to a close and start a *new* journal by copying key information (e.g. migrating) forward to encourage reflecting on what is worth moving forward.
```

I think we need to add a clause that explains why we need to do migrate forward.  I'm open to different phrasings of this.